src/core/developer.tf

```diff
@@ -0,0 +1,73 @@
+locals {
+  resource_groups_name   = format("%s-api-rg", local.project)
+  service_name           = format("%s-apim", local.project)
+  api_id                 = format("%s-ms-external-api-v1", local.project)
+  azure_apim_api_version = "2021-08-01"
+}
+
+resource "null_resource" "download_apim_external_api_v1" {
+  triggers = {
+    file_sha1 = filesha1("./api/ms_external_api/v1/open-api.yml.tpl")
+  }
+
+  depends_on = [module.apim_external_api_ms_v1]
+
+  provisioner "local-exec" {
+    command = <<EOT
+      az rest \
+        --method get \
+        --url https://management.azure.com/subscriptions/${data.azurerm_subscription.current.subscription_id}/resourceGroups/${local.resource_groups_name}/providers/Microsoft.ApiManagement/service/${local.service_name}/apis/${local.api_id} \
+        --url-parameters api-version=${local.azure_apim_api_version} export=true format=openapi \
+ 	--output-file ./env/${var.env}/developer/external/ms-external-api-v1.yaml
```

Comment:
this file is not a gitignored destination, thus this invocation would brought to a file to commit that if runned locally could be confusing.

I suggest to use the location "${path.module}/.terraform/tmp/" as base path

Reply:
@antonioT90 the _--output-file_ parameter fails if the destination directory does not exist...any suggestions?